root/components/Paginator.js

```diff
@@ -27,7 +27,7 @@ const Paginator = ({
   pager,
   pageVar = 'page',
 }: Props) => {
-  if (pager.last_page <= 1) {
+  if (pager.last_page <= 1 || !pager.last_page) {
```

Comment:
The pager JSON from these pages does *not* include last_page at all, so the paginator was always appearing regardless of whether there were only a couple events.

If this is unexpected, then we should instead try and figure out why last_page isn't always being added to the JSON.

Reply:
I think because you're not using `serialize_pager` (from lib/MusicBrainz/Server/ControllerUtils/JSON.pm) whereas other pages do.

Reply:
Oh. Duh. Will do.